packages/ogmios/src/ogmiosTxSubmitProvider.ts

```diff
@@ -0,0 +1,54 @@
+import { Buffer } from 'buffer';
+import { Cardano, TxSubmitProvider } from '@cardano-sdk/core';
+import {
+  ConnectionConfig,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
We could just disable this globally. Speaking of global configurations, it would be nice to have separate config for tests allowing explicit any and long lines. Can be addressed separately.